### PR TITLE
sfxr: init at 1.2.1

### PR DESCRIPTION
**Pull request by fgaz**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/audio/sfxr/default.nix [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+, pkgconfig
+, desktop-file-utils
+, SDL
+, gtk3
+, gsettings-desktop-schemas
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "sfxr-${version}";
```
Comment (JohnAZoidberg) [inline pkgs/applications/audio/sfxr/default.nix]: You can now use `pname = "sfxr";` and it'll create `name` itself.

File: pkgs/applications/audio/sfxr/default.nix [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    SDL
+    gtk3
+    gsettings-desktop-schemas
+    wrapGAppsHook
+  ];
+
+  makeFlags = [ "DESTDIR=$(out)" ];
+
+  meta = with stdenv.lib; {
+    homepage = http://www.drpetter.se/project_sfxr.html;
```
Comment (JohnAZoidberg) [inline pkgs/applications/audio/sfxr/default.nix]: We're probably going to deprecate unquoted URLs soon, if RFC 45 is accepted.

**Comment by JohnAZoidberg** [comment]
Pretty cool program :D Everything seems to work: Creating, playing audio, modifying all kinds of sliders, exporting, importing.

File: pkgs/top-level/all-packages.nix [hunk 3]
```diff
@@ -19859,6 +19859,8 @@ in
 
   setbfree = callPackage ../applications/audio/setbfree { };
 
+  sfxr = libsForQt5.callPackage ../applications/audio/sfxr { };
```
Comment (JohnAZoidberg) [inline pkgs/top-level/all-packages.nix]: Why do you use `libsForQt5.callPackage`?

Comment (fgaz) [reply pkgs/top-level/all-packages.nix]: I copypasted the sfxr-qt line just below but forgot to change that part. Good catch!

**Comment by aanderse** [comment]
@JohnAZoidberg thanks for your review :+1: 
@fgaz thanks for your contribution to nixpkgs! :tada:

**Pull request merged**